### PR TITLE
bump container cap to 100 for jenkins 1 and 2

### LINKED ISSUE #637: container cap in older versions
Would y'all be okay bumping the container cap in 3.7 and 3.9 since those are the most used versions right now?

### PR DESCRIPTION
**Pull request by Gl4di4torRr**
Fixes #637

### CODE REVIEW

File: 1/contrib/jenkins/kube-slave-common.sh [hunk 1]
```diff
@@ -140,7 +140,7 @@ function generate_kubernetes_config() {
       <jenkinsUrl>http://${JENKINS_SERVICE_HOST}:${JENKINS_SERVICE_PORT}</jenkinsUrl>
       <jenkinsTunnel>${JNLP_HOST}:${JNLP_PORT}</jenkinsTunnel>
       <credentialsId>1a12dfa4-7fc5-47a7-aa17-cc56572a41c7</credentialsId>
-      <containerCap>10</containerCap>
+      <containerCap>100</containerCap>
```
Comment (gabemontero) [inline 1/contrib/jenkins/kube-slave-common.sh]: don't bother for the v1 version, we are no longer publishing those images

Comment (Gl4di4torRr) [reply 1/contrib/jenkins/kube-slave-common.sh]: okay, i did it just because it was there... i'll revert the change. thanks!

Comment (gabemontero) [reply 1/contrib/jenkins/kube-slave-common.sh]: reasonable assumption on your part for sure :-)

just another snowflake of ours in this space

**Pull request merged**